geometry_msgs/Point location
int32 quality
int32 computation_time